src/components/application_manager/src/commands/hmi/get_urls.cc

```diff
@@ -194,26 +194,30 @@ void GetUrls::ProcessPolicyServiceURLs(const policy::EndpointUrls& endpoints) {
 
   // Will use only one URL for particular application if it will be found
   // Otherwise URL from default section will used
-  SmartObject service_info = SmartObject(SmartType_Map);
+  SmartObject service_infos = SmartObject(SmartType_Array);
 
+  size_t count_of_services = 0;
   for (size_t e = 0; e < endpoints.size(); ++e) {
```

Comment:
@VVeremjova please use iterators if possible, or at least use 'i' for indexing instead of 'e'